README.md

```diff
@@ -1 +1,68 @@
-# CSTATE Health Checker
+# CSTATE Health Checker Module
+
+## Descripton
+
+Provides a mechanizm for monitoring the health of a system and automatically create and resolve CSTATE issues.
+
+## Ussage
+
+### Templates
+
+The `alarm_name` property of a healthcheck must match the part of the issue template. An issue template must be located on the `/content/issues` directory and named accordingly:
+
+* first-latest.md.template
+* second-latest.md.template
+
+The template contents must be of the follwoing format
+```
+---
+section: issue
+title: An issue title of your choice
+date: {{start_date}}
+resolved: {{resolved}}
+informational: false
+resolvedWhen: {{end_date}}
+affected: ['first']
```

Comment:
I'd clarify that the `affected` field does not need to match the filename/alarm name.  The `affected` field must match the `Component` name from the cstate config.  Maybe put `['First Component']` to try and clarify?